### DISCUSSION TITLE
Does not work, CF detect me

### DISCUSSION THREAD

**Discussion by dxdate** [ultrafunkamsterdam/undetected-chromedriver#1466] [Q&A]
`import undetected_chromedriver as webdriver

def main():
    options = webdriver.ChromeOptions()
    driver = webdriver.Chrome(use_subprocess=False, options=options)
    driver.get(urls.url_ns)
    time.sleep(100)

if __name__ == "__main__":
    main()`
I am using the latest version of uc and selenium
![image](https://github.com/ultrafunkamsterdam/undetected-chromedriver/assets/75077484/e85dc47f-0413-4490-8675-20f2b00db555)
the site is sent for a reboot and again asks to confirm that I am a person

**Comment by henzycuong1** [comment]
using chrome version 102 you can pass it, but not 100% working
